mne/io/pick.py

```diff
@@ -60,7 +60,11 @@ def channel_type(info, idx):
     elif kind == FIFF.FIFFV_SYST_CH:
         return 'syst'
     elif kind in [FIFF.FIFFV_SEEG_CH, 702]:  # 702 was used before v0.11
```

Comment:
Oh yeah. We're working for 12 right now. I forgot. I think the check for 702 can be removed now.